### PR TITLE
UML diagrams

### PR DESCRIPTION
**Pull request by sushmangupta**

### CODE REVIEW

File: concepts/commerce/checkout-concept/cart.md [hunk 1]
```diff
@@ -105,7 +140,28 @@ A default set of collectors is implemented in Shopware 6, which has a set call o
 | Shopware\Core\Checkout\Promotion\Cart\CartPromotionsCollector | Enrich add, remove and validate promotions |
 | Shopware\Core\Checkout\Shipping\Cart\ShippingMethodPriceCollector | Handle shipping prices |
 
-![Cart enrichment steps](../../../assets/cart-enrichtment-steps.png)
+```mermaid
```
Comment (elkmod) [inline concepts/commerce/checkout-concept/cart.md:143]: It looks compared to the old diagram, like the *Collect* step is missing

File: concepts/framework/flow-concept.md [hunk 2]
```diff
@@ -43,8 +43,35 @@ In Shopware, you have multiple interfaces and classes for different types of eve
 
 Once the action on the Storefront or from the app happens, the FlowDispatcher will dispatch FlowEventAware to the FlowExecutor. From here, the FlowExecutor will check the condition to decide whether to execute the action.
 
-![Flow builder concept for flow sequence](../../assets/flow-concept-1.png)
+```mermaid
```
Comment (elkmod) [inline concepts/framework/flow-concept.md:46]: GitHub has difficulties displaying the rich diff here (i.e. rendering the mermaid). We should check after the deployment if it is rendered correctly.

File: concepts/framework/rules.md [hunk 3]
```diff
@@ -39,11 +39,32 @@ The center of the rule system is the `Rule`. It is realized as a variant of the
 
 As well as a Specification class, a Rule class represents a condition to fulfill. It implements the `match(RuleScope $scope)` function to validate user defined values against a runtime state. See the following object diagram for a better understanding:
 
-![](../../assets/rule-objects.png)
+```mermaid
```
Comment (elkmod) [inline concepts/framework/rules.md:42]: Same preview error here.

**Review by elkmod — CHANGES_REQUESTED** [review]
Please remove all the images that are not used anymore.

File: concepts/commerce/checkout-concept/cart.md (hunk 1 above)
Comment (sushmangupta) [reply concepts/commerce/checkout-concept/cart.md:143]: I was asked to remove it by checkout team

**Review by Isengo1989 — APPROVED** [review]

**Pull request merged**